examples/jupyter/expansion-filters.ipynb

```diff
@@ -437,6 +439,1116 @@
    "source": [
     "In addition to being able to tally Legendre moments, there are also functional expansion filters available for spherical harmonics (`SphericalHarmonicsFilter`) and Zernike polynomials over a unit disk (`ZernikeFilter`). A separate `LegendreFilter` class can also be used for determining Legendre scattering moments (i.e., an expansion of the scattering cosine, $\\mu$)."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, Zernike polynomials provided in OepnMC to represent continuous functions of space or angle can be reconstructed from the tallied moments.\n",
```

Comment:
This is a little redundant since the sentence above (from existing notebook) just mentioned that a zernike filter can be used. I would just start off by saying something like "Now let's look at an example of functional expansion tallies using Zernike polynomials as the basis functions." Also, to demarcate this portion of the notebook, I would use markdown in a text cell to add a section heading using `##`:
```md
## Zernike polynomials

Now let's look at an example ...
```
When the notebook is rendered, the title will appear in a `<h2>` html block.